docker/staging/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.3'
+services:
+  coordinator:
+    build:
+      context: "../../"
+      dockerfile: "docker/Dockerfile"
+    networks:
+      - labgrid_net
+    tty: true
+    command: "crossbar start"
```

Comment:
The coordinator will write persistent data to it's working directory (a yaml file of the configured places). Is that kept over docker container restarts?